.circleci/config.yml

```diff
@@ -81,8 +81,8 @@ jobs:
           root: .
           # Must be relative path from root
           paths:
-            - dist/htmxl-0.8.5.tar.gz
-            - dist/htmxl-0.8.5-py3-none-any.whl
```

Comment:
We updated to 0.8.6 a few months ago, which resulted in this causing CI to always fail as there would be no packages being built for v0.8.5. I think a `*` is safe here and should prevent that silly CI failure in the future.